packages/block-editor/src/components/block-media-update-progress/test/index.native.js

```diff
@@ -0,0 +1,501 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * WordPress dependencies
+ */
+import { sendMediaUpload, sendMediaSave } from '@wordpress/react-native-bridge';
+
+/**
+ * Internal dependencies
+ */
+import {
+	BlockMediaUpdateProgress,
+	MEDIA_UPLOAD_STATE_UPLOADING,
+	MEDIA_UPLOAD_STATE_SUCCEEDED,
+	MEDIA_UPLOAD_STATE_FAILED,
+	MEDIA_UPLOAD_STATE_RESET,
+	MEDIA_SAVE_STATE_SAVING,
+	MEDIA_SAVE_STATE_SUCCEEDED,
+	MEDIA_SAVE_STATE_FAILED,
+	MEDIA_SAVE_STATE_RESET,
+	MEDIA_SAVE_FINAL_STATE_RESULT,
+	MEDIA_SAVE_MEDIAID_CHANGED,
+} from '../';
+
+jest.mock( '@wordpress/react-native-bridge', () => {
+	const callUploadCallback = ( payload ) => {
+		this.uploadCallBack( payload );
+	};
+	const callSaveCallback = ( payload ) => {
+		this.saveCallBack( payload );
+	};
+	const subscribeMediaUpload = ( callback ) => {
+		this.uploadCallBack = callback;
+	};
+	const subscribeMediaSave = ( callback ) => {
+		this.saveCallBack = callback;
+	};
+	const mediaSources = {
+		deviceCamera: 'DEVICE_CAMERA',
+		deviceLibrary: 'DEVICE_MEDIA_LIBRARY',
+		siteMediaLibrary: 'SITE_MEDIA_LIBRARY',
+	};
+	return {
+		subscribeMediaUpload,
+		subscribeMediaSave,
+		sendMediaUpload: callUploadCallback,
+		sendMediaSave: callSaveCallback,
+		mediaSources,
+	};
+} );
+
+// const MEDIA_ID_REMOTE = 123;
+const MEDIA_ID_LOCAL = 2;
+const MEDIA_ID_TEMP = 'tempid-0-1';
```

Comment:
nitpick: `MEDIA_SAVE_MEDIAID_CHANGED` uses `MEDIAID` while this uses `MEDIA_ID`. I'd change for consistency to whichever you prefer.

Reply:
Totally up to you though of course

Reply:
good eye! Renamed const value refs in 453e0d0ea1